XCUITests/SettingsTest.swift

```diff
@@ -18,8 +18,12 @@ class SettingsTest: BaseTestCase {
 
         waitUntilPageLoad()
         waitForValueContains(app.textFields["url"], value: "support.mozilla.org")
-        waitForExistence(app.webViews.staticTexts["Firefox for iOS"])
-        XCTAssertTrue(app.webViews.staticTexts["Firefox for iOS"].exists)
+        if iPad() {
+            print (app.debugDescription)
```

Comment:
We still need this?

Reply:
no, we don't need that...sorry I forgot to remove it and did not realized...Deleted now